README.md

```diff
@@ -62,9 +62,14 @@ Zowe CLI requires NPM version 7 to install from source. Before proceeding, check
 The first time that you download Zowe CLI from the GitHub repository, issue the following command to install the required Zowe CLI dependencies and several development tools:
 
 ```
-npm install --@zowe:registry=https://registry.npmjs.org --no-package-lock --force
+npm install
 ```
 
+The flags from previous command are unnecessary now and may cause some problems, if you are wondering what problems might be caused here you go:
+- ```--@zowe:registry=https://registry.npmjs.org``` is unnecessary because the ```@zowe:registry``` defined in .npmrc should work.
+- ```--no-package-lock``` ignores the dependencies locked down in shrinkwrap for security/compatiblity reasons.
+- ``` --force ``` should generally not be used unless there is an error that requires force to be bypassed.
+
```

Comment:
Thanks for updating the command 🙂 Please remove these lines, just updating the command above is sufficient. Most people reading this won't know the context of what the previous command was.

Reply:
Okay sure, done.
I thought it would be helpful to state that there but got what you wanted to say.
Thanks